Makefile.am

```diff
@@ -257,7 +257,7 @@ endif
 VALGRIND_ARGS = --trace-children=yes --quiet --error-exitcode=33 \
 	--suppressions=valgrind-suppressions --gen-suppressions=all \
 	--num-callers=16 --leak-check=yes --show-leak-kinds=definite \
-	--errors-for-leak-kinds=definite --trace-children-skip='*mock*,/bin*,/usr/bin/*,/usr/local/bin,*cockpit-bridge,*cockpit-stub'
+	--errors-for-leak-kinds=definite --trace-children-skip='*mock*,/bin*,/usr/bin/*,/usr/local/bin,*cockpit-stub'
```

Comment:
In this case we should also take out cockpit-stub.

Reply:
Indeed, I thought that was just a test thing, but it's actually shipped. This causes quite a bunch of more errors though, so would you mind if we fix that in a separate PR?

Reply:
I don't see extra errors running with cockpit-stub. Can you run it here so we can take a look?

In general though, the main problem we've had with this is unclean shutdowns in tests where things get killed before the clean up code has had a chance to run.  Maybe we've cleaned most of that up so it's not a problem anymore.

Reply:
Sure, I filed #7753 for this, as I got this so nicely green. If that works, we can use that instead of this one, and I'll squash the commits together.